source/wecSim.m

```diff
@@ -58,6 +58,10 @@
     end; clear n combine;
 end
 % Waves and Simu: check inputs
+try 
+    waves.spectrumDataFile = ['..' filesep parallelComputing_dir filesep '..' filesep waves.spectrumDataFile];
+    waves.etaDataFile      = ['..' filesep parallelComputing_dir filesep '..' filesep waves.etaDataFile];
+end
 waves.checkinputs;

```

Comment:
Do you think we could put this in the `if exist('mcr','var') == 1` if statement on line 51?

Reply:
Yep, this works. I'm going to make the change so that this `try` is only executed for MCR runs.